BiologyAnimation/ek_cell_types/Slide16/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
```

Comment:
In one of the legacy reports,

there was instruction to not use cdn
download the bootstrap stylesheet and use

@biradarganesh25  @cerodav